solutions/usaco-784.mdx

```diff
@@ -64,4 +67,39 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+## Python Implementation
+
+```py
+file_in = open('lifeguards.in')
+data = file_in.read().strip().split('\n')
+n = int(data[0])
+
+lifeguards = [list(map(int, data[i + 1].split(' '))) for i in range(n)]
+
+MAX_TIME = 1000
```

Comment:
put this line at the top of the file

Reply:
I'd say it's personal preference, so probably not necessary

Reply:
by convention constants are at the top i think